.github/workflows/tests-arm64.yaml

```diff
@@ -35,6 +35,7 @@ jobs:
         run: |
           set -euo pipefail
           go clean -testcache
+          make gofail-enable
```

Comment:
Note, not sure if we want to enable gofail in unit tests.

Reply:
Thanks for reminder! I overlook it. Updating..